.changeset/sixty-suits-try.md

```diff
@@ -0,0 +1,6 @@
+---
+'@keystonejs/test-utils': major
+---
+
+- Removed some of the redundant functions like `findOne`, `findById` from `_keystoneRunner` function. Instead, using utils functions from `server-side-graphql-client` package.
```

Comment:
Let's be really explicit and list exactly which functions where removed. If I'm a user of this package I want to know exactly what's gone away so I can assess whether this will break my code.